ruby/vampires.rb

```diff
@@ -32,16 +36,20 @@
     health_insurance = true 
   end 
 
-  until employee_allergy == "sunshine"
-    puts "Please list your allergies. When you are finished, type 'done'."
+  puts "Please list your allergies. When you are finished listing your allergies, type 'done'."
+  until employee_allergy == "sunshine" || employee_allergy == "done"
+
     employee_allergy = gets.chomp
   end
 
   if employee_allergy == "sunshine"
     puts "Probably a vampire."
+
+  else
+    puts "Results inconclusive."
   end
 
-  if vampire_name == "Drake Cula" || vampire_name == "Tu Fang")
+  if vampire_name == "Drake Cula" || vampire_name == "Tu Fang"
```

Comment:
This looks good and below is merely a design suggestion.

Lets consider order of operations here.  I understand that the instructions may be guiding you to just put in conditions as they are mentioned.  However, we could clean this up and make the file much more efficient by analyzing our conditions for optimal calls and runtime.

If the employee is named "Drake Cula" or "Tu Fang" then we know that they are definitely a vampire. If that's the case, is there any need to even run the rest of the code?

Consider nesting your conditions and creating a strong order of operations for optimized logic and runtime.

```
if name == "Drake Cula" || name == "Tu Fang"
   "definitely a vampire"
else
  if age_check
    do something
  elsif garlic_and_insurance_check
    do something
  else
    do something
  end
end
```